### PR TITLE
scaffolding for 29-fee

### PR DESCRIPTION
**Pull request by colin-axner**
## Description

closes: #260 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: modules/apps/29-fee/keeper/genesis.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package keeper
+
```
Comment (AdityaSripal) [inline modules/apps/29-fee/keeper/genesis.go:2]: uncomment

File: modules/apps/29-fee/keeper/grpc_query.go [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package keeper
+
+// TODO
```
Comment (AdityaSripal) [inline modules/apps/29-fee/keeper/grpc_query.go:3]: uncomment

File: modules/apps/29-fee/keeper/keeper.go [hunk 3]
```diff
@@ -0,0 +1,87 @@
+package keeper
+
+/*
```
Comment (AdityaSripal) [inline modules/apps/29-fee/keeper/keeper.go:3]: uncomment

File: modules/apps/29-fee/keeper/keeper_test.go [hunk 4]
```diff
@@ -0,0 +1,45 @@
+package keeper_test
+
+/*
```
Comment (AdityaSripal) [inline modules/apps/29-fee/keeper/keeper_test.go:3]: uncomment

File: modules/apps/29-fee/keeper/msg_server.go [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package keeper
+
+// TODO
+//var _ types.MsgServer = Keeper{}
```
Comment (AdityaSripal) [inline modules/apps/29-fee/keeper/msg_server.go:4]: uncomment

File: modules/apps/29-fee/module_test.go [hunk 6]
```diff
@@ -0,0 +1,244 @@
+package fee_test
```
Comment (AdityaSripal) [inline modules/apps/29-fee/module_test.go:1]: Remove this file since it will be very customized

**Review by AdityaSripal — CHANGES_REQUESTED** [review]
uncomment code files and remove test files

**Review by seantking — APPROVED** [review]

**Review by AdityaSripal — APPROVED** [review]

**Pull request merged**